Client/Frontend/Browser/Onboarding/onboarding-rewards.json

```diff
@@ -1 +1 @@
,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[300,300],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Path","np":2,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":190.000007738859,"st":0,"bm":0},{"ddd":0,"ind":26,"ty":4,"nm":"panel-tip","sr":1,"ks":{"o":{"a":1,"k":[{"i":{"x":[0.667],"y":[1]},"o":{"x":[0.167],"y":[0.167]},"t":78,"s":[100]},{"t":83.0000033806593,"s":[0]}],"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":1,"k":[{"i":{"x":0.667,"y":1},"o":{"x":0.333,"y":0},"t":15,"s":[804.264,431.489,0],"to":[4.667,70.833,0],"ti":[-4.667,-70.833,0]},{"t":20.0000008146167,"s":[832.264,856.489,0]}],"ix":2},"a":{"a":0,"k":[0,0,0],"ix":1},"s":{"a":1,"k":[{"i":{"x":[0.667,0.667,0.667],"y":[1,1,1]},"o":{"x":[0.333,0.333,0.333],"y":[0,0,0]},"t":15,"s":[0,0,100]},{"t":20.0000008146167,"s":[100,100,100]}],"ix":6}},"ao":0,"shapes":[{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":0,"k":{"i":[[0.33,-0.34],[0,-0.58],[-0.32,-0.34],[-0.5,0],[-0.31,0.34],[0,0.6],[0.31,0.34],[0.51,0]],"o":[[-0.32,0.34],[0,0.58],[0.33,0.35],[0.51,0],[0.31,-0.33],[0,-0.59],[-0.31,-0.33],[-0.5,0]],"v":[[5.702,-1.527],[5.221,-0.137],[5.702,1.253],[6.941,1.773],[8.182,1.264],[8.652,-0.137],[8.182,-1.536],[6.941,-2.036]],"c":true},"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ind":1,"ty":"sh","ix":2,"ks":{"a":0,"k":{"i":[[-0.51,-0.32],[-0.3,-0.6],[0,-0.82],[0.29,-0.61],[0.51,-0.32],[0.65,0],[0.43,0.27],[0.2,0.46],[0,0],[0,0],[0,0],[0,0],[0,0],[-0.43,0.27],[-0.6,0]],"o":[[0.51,0.32],[0.29,0.6],[0,0.82],[-0.3,0.61],[-0.51,0.33],[-0.6,0],[-0.43,-0.27],[0,0],[0,0],[0,0],[0,0],[0,0],[0.2,-0.46],[0.43,-0.27],[0.65,0]],"v":[[9.441,-3.656],[10.652,-2.266],[11.092,-0.137],[10.652,1.993],[9.441,3.383],[7.712,3.874],[6.171,3.464],[5.232,2.363],[5.232,7.493],[2.842,7.493],[2.842,-4.036],[5.232,-4.036],[5.232,-2.626],[6.171,-3.727],[7.712,-4.137]],"c":true},"ix":2},"nm":"Path 2","mn":"ADBE Vector Shape - Group","hd":false},{"ind":2,"ty":"sh","ix":3,"ks":{"a":0,"k":{"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[-1.764,3.771],[-1.764,-4.038],[0.626,-4.038],[0.626,3.771]],"c":true},"ix":2},"nm":"Path 3","mn":"ADBE Vector Shape - Group","hd":false},{"ind":3,"ty":"sh","ix":4,"ks":{"a":0,"k":{"i":[[-0.26,0.23],[-0.43,0],[-0.26,-0.24],[0,-0.36],[0.26,-0.24],[0.43,0],[0.26,0.24],[0,0.36]],"o":[[0.26,-0.24],[0.43,0],[0.26,0.23],[0,0.36],[-0.26,0.24],[-0.43,0],[-0.26,-0.24],[0,-0.36]],"v":[[-1.598,-7.137],[-0.559,-7.497],[0.471,-7.137],[0.872,-6.236],[0.471,-5.346],[-0.559,-4.986],[-1.598,-5.346],[-1.988,-6.236]],"c":true},"ix":2},"nm":"Path 4","mn":"ADBE Vector Shape - Group","hd":false},{"ind":4,"ty":"sh","ix":5,"ks":{"a":0,"k":{"i":[[0,0],[0,0],[0,0],[0,0],[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,
```

Comment:
?

Reply:
This is part of the ticket. https://github.com/brave/brave-rewards-ios/issues/239